<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStuInfoDlg</class>
 <widget class="QDialog" name="AddStuInfoDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加/修改</string>
  </property>
  <widget class="QPushButton" name="btnConfirm">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>251</y>
     <width>39</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>gpa：</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spbGpa">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>251</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>41</y>
     <width>45</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>学号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtId">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>41</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtName">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>81</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>81</y>
     <width>45</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>121</y>
     <width>45</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>性别：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbSex">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>121</y>
     <width>94</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>94</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>162</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>手机号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtPhone">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>162</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>212</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>cet4:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spbCet4">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>212</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>0</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
